zap/src/main/java/org/zaproxy/zap/extension/spider/SpiderScan.java

```diff
@@ -443,9 +454,7 @@ public void foundURI(String uri, String method, FetchStatus status) {
     }
 
     @Override
-    public void run() {
-        // TODO Auto-generated method stub
-    }
+    public void run() {}
```

Comment:
You must not have installed SonarLint :wink:

It would have asked you to address the TODO. When you removed the TODO it would have asked you to add a useful comment for the blank method body.

Reply:
Arg! OK OK, I'll look at installing it!

Reply:
&lt;grin&gt;

Reply:
It should have underlined it in blue. But if it's a fresh install maybe it's still crunching through things.

Reply:
Like this:
![image](https://user-images.githubusercontent.com/7570458/133296822-abfb2ef5-fe26-41e8-ad1f-5fec6cfeabae.png)